java/rocksjni.pom

```diff
@@ -19,6 +19,11 @@
             <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
             <distribution>repo</distribution>
         </license>
+        <license>
+            <name>GNU General Public License, version 2</name>
+            <url>http://opensource.org/licenses/gpl-2.0</url>
```

Comment:
The one we are using is https://www.gnu.org/licenses/gpl-2.0.txt
Maybe I should put that one.